packages/dev/core/src/Misc/textureTools.ts

```diff
@@ -285,82 +282,9 @@ const ProcessAsync = async (
         scene.postProcessManager.directRender([lodPostProcess], rtt.renderTarget, true);

```

Comment:
You have to call `engine.restoreDefaultFramebuffer();` because the default frame buffer is not restored automatically when calling `directRender`.

Reply:
@Popov72 , double checking on this: The framebuffer is already unbound with `engine.unBindFramebuffer(rtt.renderTarget);`.

I looked at the restoreDefaultFramebuffer() implementation and it does some cache wiping too-- seems more overhead maybe?

Reply:
Yes, my bad, I didn't see the `unBindFramebuffer` below!

Reply:
Ok, I'll keep it as is